ministryofjustice/correspondence_tool_staff
# CT-2614 Add liveness / readiness probes to webapp

## Description
Add liveness and readiness probes for Kubernetes to deployment files only.

## Self-review checklist


### Screenshots
n/a

### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2614

### Deployment
Need to deploy to all envs, prod last after satisfactory testing on other envs.

### Manual testing instructions
deploy last tag, or build a new one then deploy to development. Check if site is working. Need to do same for other envs.
